src/Extra.php

```diff
@@ -0,0 +1,496 @@
+<?php
+
+/**
+ * Automatically generated
+ */
+
+namespace Webmozart\Assert;
+
+class Extra extends Assert
```

Comment:
Probably better to generate a trait that integrates with `Assert`. The rationale is that we don't want to modify the file in-place

Reply:
We wouldn't modify it in place - we would extend it the way I did. But yep, trait could be a little bit nicer.